andela/boondocks-bn-backend
# #169257416 Fix the Profile and Comment tests bug

#### What does this PR do?
Fixes the Profile and Comment tests bug, which was causing to other tests to fail.
#### Description of Task to be completed?
- Wrapped the `before` hook of the test inside the `describe` section
#### How should this be manually tested?
- Clone the repo to the local machine
- Run `npm run pretest-travis` to migrate required tables
- Run `npm run test`
All test will run well as expected.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
